CMakeLists.txt

```diff
@@ -6,7 +6,13 @@ find_package(ament_cmake REQUIRED)
 
 include_directories(include)
 
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
 if(NOT WIN32)
+  add_definitions(-D_POSIX_C_SOURCE=199309L)
```

Comment:
This was needed for `clock_gettime`, but only if `std=c99` was used instead. `CMAKE_C_STANDARD` already causes CMake to set the appropriate flags so `_POSIX_C_SOURCE` is not needed